### DISCUSSION TITLE
Custom fiels on devices

### DISCUSSION THREAD

**Discussion by JulienMartelet** [netalertx/NetAlertX#871] [General]
Hello, 
Is it possible to add a field to a machine file ? A field with a clickable public URL.

**Comment by jokob-sk** [comment]
Hi, not currently. Can you please explain your use case in more detail? What alternatives did you consider. How would you use it?

**Reply by JulienMartelet** [reply]
"Hello,
The goal of adding a field with a clickable public URL would be to allow quick access to certain web services for my personal network machines. Currently, I use a PHP web application with a MariaDB database that I developed myself. After installing and using NetAlertX, I found it very well-developed. Adding this field would centralize all the information in one place. Please let me know if this feature might be possible in the future."

**Comment by JulienMartelet** [comment]
![2024-11-04 20_49_56-(137) NetAlertX - NAS-HAUTCHAP4](https://github.com/user-attachments/assets/30bb400b-597a-4f92-bbf5-d64d27982554)

**Comment by JulienMartelet** [comment]
Hello,

I tested netalertx-dev in new instance in parallel with my netalertX of production, but i have a problem to load information of device from db.
I can't testing the new functionality HTTPS field.

I have update my netalertx of production, I feel like the HTTPS field functionality is not implemented.

See error in attachment =>

![2025-01-02 11_56_53-(1) NetAlertX - NAS-HAUTCHAP4](https://github.com/user-attachments/assets/6852b7a1-efb0-4637-9550-c506643f6703)

**Reply by jokob-sk** [reply]
Hi,

Please make sure you use a unique GRAPHQL_PORT value across your instances. Looks like this instance is trying to connect to another instance.

Hope this helps,
J

**Comment by JulienMartelet** [comment]
It's ok now, for my dev environement. What is the name of new field with web link ? (see attachment)
And where i can see the version of NetAlertx in web interface ?

![2025-01-02 16_07_53-(1) NetAlertX - NAS-HAUTCHAP4](https://github.com/user-attachments/assets/f4aea5db-6725-419c-bef3-9936604c57e5)

**Reply by jokob-sk** [reply]
It's called Props/actions and docs can be found here : https://github.com/jokob-sk/NetAlertX/blob/main/docs/CUSTOM_PROPERTIES.md

**Comment by JulienMartelet** [comment]
Yes !!! it's perfect. The new features are very well thought out and practical. :-)

![2025-01-02 21_19_43-(2) NetAlertX - NAS-HAUTCHAP4](https://github.com/user-attachments/assets/130ff35b-ad20-408c-b20b-67527b4c634a)

**Reply by jokob-sk** [reply]
Gladys to hear that 👍🙂
